css-dev/burf-theme/widgets/_widgets.scss

```diff
@@ -139,6 +139,12 @@ $border-widget-list:                       $border !default;
 		&:focus {
 			color: $color-widget-link-hover;
 		}
+
+		&.button-primary,
+		&.button-primary:hover,
+		&.button-primary:focus {
+			color: $color-grayscale-f;
```

Comment:
This variable will work now, but what if someone sets the primary button color background to white? I would change this to use the primary button color variable instead: https://bu-ist.github.io/responsive-foundation/2.0.0-dev/#01-config-variable-color-button-primary-text

On that note - there is also a normal button in addition to the primary button. We should probably add that rule here as well - we'll encounter the same bug for the same reasons.